yarn-project/ethereum/src/l1_tx_utils.ts

```diff
@@ -405,14 +419,28 @@ export class L1TxUtils {
     // Ensure priority fee doesn't exceed max fee
     const maxPriorityFeePerGas = priorityFee > maxFeePerGas ? maxFeePerGas : priorityFee;
 
+    if (attempt > 0 && previousGasPrice?.maxFeePerBlobGas) {
+      const bumpPercentage =
+        gasConfig.priorityFeeRetryBumpPercentage! > MIN_REPLACEMENT_BUMP_PERCENTAGE
+          ? gasConfig.priorityFeeRetryBumpPercentage!
+          : MIN_REPLACEMENT_BUMP_PERCENTAGE;
+
+      blobBaseFee = (previousGasPrice.maxFeePerBlobGas * (100n + bumpPercentage)) / 100n;
```

Comment:
Why do we use the `priorityFeeRetryBumpPercentage` here? I'd expect that we just need to use the `MIN_REPLACEMENT_BUMP_PERCENTAGE` [expected by geth](https://github.com/ethereum/go-ethereum/blob/a9ab53d751b018ee493b909d520eb5f3daba6734/core/txpool/blobpool/blobpool.go#L1136-L1173) (which comes from [here](https://github.com/ethereum/go-ethereum/blob/a9ab53d751b018ee493b909d520eb5f3daba6734/cmd/utils/flags.go#L408C13-L408C50) apparently).

Also, we need to account for the current blob base fee, and do something similar to what we do for the calldata fee: use the minimum between the current chain blob base fee (bumped by 12.5% per stall time) and geth's minimum bump (10%) wrt the previous price.

Reply:
thanks for that 🙏 
I was googling to find if there was such a percentage but couldn't find it so I used this patchy solution.
I should go directly to geth code from now on